src/row.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+const Row = ({ element, index }) => {
+  const [active, setActiveClass] = useState(false);
+  function handleClick() {
+    setActiveClass(!active);
+  }
```

Comment:
This is really excellent work @ellietms! Could you explain why this works?

Reply:
@40thieves  yes, the value of active is false when someone clicks in the rows of the table it is going to change to true and add the classes

Reply:
Yep that's correct 🙂 

Another question for you: why do you think it's necessary to have a `Row` component?

Reply:
@40thieves at first  I Had all of my codes inside of search result component but when I clicked in one row of the table all of the rows changed the colour I really don't know why but when I changed them to another component it worked!

Reply:
Yep, that's exactly the problem. We'll talk more about it on Sunday

Reply:
@40thieves  thank you ali